gitlab/v4/objects/commits.py

```diff
@@ -202,10 +202,10 @@ def create(
         # project_id and commit_id are in the data dict when using the CLI, but
         # they are missing when using only the API
         # See #511
-        base_path = "/projects/%(project_id)s/statuses/%(commit_id)s"
+        base_path = base_path = "/projects/{project_id}/statuses/{commit_id}"
```

Comment:
This looks wrong to me, or at least a redundant `=` sign.

Reply:
Same as above 🤦 thanks